src/controllers/update/overseas.entity.update.details.controller.ts

```diff
@@ -52,10 +53,16 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     logger.debugRequest(req, `POST OVERSEAS_ENTITY_UPDATE_DETAILS`);
 
     const data: ApplicationDataType = mapRequestToEntityData(req);
-
     const session = req.session as Session;
+    const entityName = req.body[EntityNameKey];
+
     setApplicationData(session, data, EntityKey);
 
+    setExtraData(req.session, {
+      ...getApplicationData(req.session),
+      [EntityNameKey]: entityName
+    });
+
     logger.debugRequest(req, `POST ${config.OVERSEAS_ENTITY_REVIEW_PAGE}`);
```

Comment:
Is this just a repeat of line 53?

Reply:
It isn't, the values are different. This one looks to logging for the redirect happening on the next line... I've removed it since the controller for that will handle it's own logging. Guess it's just an error from before.